### ISSUE TITLE
false positive messages with using of github.com/pkg/errors

### ISSUE THREAD

**Issue by Dentrax** [tomarrell/wrapcheck#12]
This code causes a _false-positive_ error:

```go
import "github.com/pkg/errors"


func FooFunc() err {
 b, err := BarFunc()
 if err != nil {
  return errors.Wrap(err, "this line causes: error returned from external package is unwrapped")
 }
}

func BarFunc() (bool, err) {
 // ...
}
```

Version: `golangci-lint has version 1.39.0 built from 9aea4aee on 2021-03-26T08:02:53Z`

**Comment by tomarrell** [comment]
Hi Dentrax,

This has already been resolved in the `v1.1.0` release of this linter, specifically covering `github.com/pkg/errors`. The update has been merged into master of golangci-lint as well and will be released in the `v1.40.0`.

Cheers
